src/stt-settings/credentials.js

```diff
@@ -1,28 +1,93 @@
-const fs = require('fs');
-const path = require('path');
-const electron = require('electron');
-const appUserDataPath = electron.remote.app.getPath('userData');
+const fs = require("fs");
+const path = require("path");
+const electron = require("electron");
+const downloadDeepSpeechModel = require("deepspeech-node-wrapper")
+  .downloadDeepSpeechModel;
+const appUserDataPath = electron.remote.app.getPath("userData");
+// TODO: consider moving deepspeech logic to a separate file from credentials.js?
+
+function getDeepSpeechModelFolderName( modelVersion) {
```

Comment:
```suggestion
function getDeepSpeechModelFolderName( modelVersion = "0.6.0") {
```

One way to get default params in easily